config/routes.rb

```diff
@@ -33,14 +33,14 @@
     get 'i/signout', action: :logout, as: :newflow_logout
   end
 
-  scope controller: 'newflow/signup' do
+  scope controller: 'signup' do
```

Comment:
I think this will allow us to do a blanket match on `i` urls and redirect them with params with a nice function-isk thing!